.sanity-ansible-ignore-2.13.txt

```diff
@@ -0,0 +1,77 @@
+plugins/module_utils/network_lsr/nm/__init__.py empty-init!skip
```

Comment:
just wondering: Is the filename ".sanity-ansible..." required by an Ansible project or is it a filename pattern that tox-lsr uses? It would  be nice to not have many files with the same or very similar content.

Reply:
> just wondering: Is the filename ".sanity-ansible..." required by an Ansible project or is it a filename pattern that tox-lsr uses?

It is a filename pattern used by the tools that build the collection - https://github.com/linux-system-roles/auto-maintenance/blob/master/release_collection.py#L227 - we use these to build the `tests/sanity/ignore-xx.txt` files required by Ansible.

> It would be nice to not have many files with the same or very similar content.

There are version differences - for example, the 2.14 file must contain ignores for python-3.11, but earlier versions must not have the ignores for python-3.11.  So yes, we could have
* a config file that lists the ignores and suppressions in a more compact format
* a script that generates the correct ignore-xxx.txt format for each version
* a workflow to ensure that files are generated properly (e.g. git post-commit, github actions)

But at the moment that seems like a lot of work . . . otoh this would also be used by other roles that have to do something similar - I'll create a jira task to investigate doing this

Reply:
I just noticed that the 2.11 file also contains ignores for python-3.11, how can we exactly know the version difference ?
 (e.g. ignores for python-3.11 )